### PR TITLE
Fix a lot of English typos

### PR DESCRIPTION
**Pull request by CrafterKolyan**

### CODE REVIEW

**Comment by huneau** [comment]
nice job :)

**Comment by huneau** [comment]
@Irgendwer01 Can I merge?

**Comment by Irgendwer01** [comment]
@huneau @CrafterKolyan Does the Configs got tested? Because it could be that the Configs will just be replaced at load because of the changes

**Comment by CrafterKolyan** [comment]
> @huneau @CrafterKolyan Does the Configs got tested? Because it could be that the Configs will just be replaced at load because of the changes

I don't think it does really matter. There are not a lot of people looking at the configs of modpack.

**Comment by CrafterKolyan** [comment]
And actually comments in configs are not looked at Minecraft at all. Even if the comment in mod is different nothing will change and everything will be working.

**Comment by huneau** [comment]
@Irgendwer01 No idea if the config is reload if comment are updated

File: config/exnihilocreatio/ExNihiloCreatio.cfg [hunk 1]
```diff
@@ -113,7 +113,7 @@ exnihilocreatio {
         # Barrel lighting can change during transformations
         B:enableBarrelTransformLighting=true
 
-        # Enable this to to load the JSON files.
```
Comment (brachy84) [inline config/exnihilocreatio/ExNihiloCreatio.cfg]: please, undo this change

File: config/forestry/common.cfg [hunk 2]
```diff
@@ -98,7 +98,7 @@ power {
     }
 
     display {
-        # The format power will be displayed in in engine/machine interfaces. [default: RF] [valid: [RF, FE, MJ, TESLA]]
```
Comment (brachy84) [inline config/forestry/common.cfg]: undo this change

File: config/opencomputers/settings.conf [hunk 3]
```diff
@@ -415,7 +415,7 @@ opencomputers {
     verbosePersistenceErrors=false
   }
 
-  # File system related settings, performance and and balancing.
```
Comment (brachy84) [inline config/opencomputers/settings.conf]: also undo this

File: config/opencomputers/settings.conf [hunk 4]
```diff
@@ -466,7 +466,7 @@ opencomputers {
     # The maximum block size that can be read in one 'read' call on a file
     # system. This is used to limit the amount of memory a call from a user
     # program can cause to be allocated on the host side: when 'read' is,
-    # called a byte array with the specified size has to be allocated. So if
```
Comment (brachy84) [inline config/opencomputers/settings.conf]: and this

File: config/opencomputers/settings.conf [hunk 5]
```diff
@@ -592,7 +592,7 @@ opencomputers {
 
     # This is a list of blacklisted domain names. If an HTTP request is made
     # or a socket connection is opened the target address will be compared
-    # to the addresses / address ranges in this list. It it is present in this
```
Comment (brachy84) [inline config/opencomputers/settings.conf]: and this

File: config/opencomputers/settings.conf [hunk 6]
```diff
@@ -719,7 +719,7 @@ opencomputers {
 
     # The probability (or rather, weighted chance) that a program disk is
     # spawned as loot in a treasure chest. For reference, iron ingots have
-    # a value of 10, gold ingots a value of 5 and and diamonds a value of 3.
```
Comment (brachy84) [inline config/opencomputers/settings.conf]: and this

File: config/opencomputers/settings.conf [hunk 7]
```diff
@@ -950,7 +950,7 @@ opencomputers {
       # The amount of bonus energy a capacitor can store for each other
       # capacitor it shares a face with. This bonus applies to both of the
       # involved capacitors. It reaches a total of two blocks, where the
-      # bonus is halved for the second neighbor. So three capacitors in a
```
Comment (brachy84) [inline config/opencomputers/settings.conf]: and this

File: config/opencomputers/settings.conf [hunk 8]
```diff
@@ -1018,7 +1018,7 @@ opencomputers {
     # The amount of energy a Charger transfers to each adjacent robot per tick
     # if a maximum strength redstone signal is set. Chargers load robots with
     # a controllable speed, based on the maximum strength of redstone signals
-    # going into the block. So if a redstone signal of eight is set, it'll
```
Comment (brachy84) [inline config/opencomputers/settings.conf]: and this

File: config/opencomputers/settings.conf [hunk 9]
```diff
@@ -1295,7 +1295,7 @@ opencomputers {
     }
 
     # The energy efficiency of the solar generator upgrade. At 1.0 this will
-    # generate as much energy as you'd get by burning  fuel in a BuildCraft
```
Comment (brachy84) [inline config/opencomputers/settings.conf]: and this

File: config/opencomputers/settings.conf [hunk 10]
```diff
@@ -1561,7 +1561,7 @@ opencomputers {
       baseValue=50
 
       # This is the amount of additional energy that fits into a robots
-      # internal buffer for each level it gains. So with the default values,
```
Comment (brachy84) [inline config/opencomputers/settings.conf]: and this

File: config/opencomputers/settings.conf [hunk 11]
```diff
@@ -1593,7 +1593,7 @@ opencomputers {
 
       # The additional "efficiency" a robot gains in using tools with each
       # level. This basically increases the chances of a tool not losing
-      # durability when used, relative to the base rate. So for example, a
```
Comment (brachy84) [inline config/opencomputers/settings.conf]: and this

File: config/rftools/rftools.cfg [hunk 12]
```diff
@@ -40,7 +40,7 @@ booster {
     # Maximum RF storage that the booster can hold [range: 0 ~ 2147483647, default: 200000]
     I:boosterMaxRF=200000
 
-    # RF per tick that the the booster can receive [range: 0 ~ 2147483647, default: 1000]
```
Comment (brachy84) [inline config/rftools/rftools.cfg]: and this

File: config/rftools/rftools.cfg [hunk 13]
```diff
@@ -286,7 +286,7 @@ environmental {
     # The minimum RF/tick usage that an active controller consumes [range: 0 ~ 2147483647, default: 5]
     I:environmentalMinRFUsage=5
 
-    # RF per tick that the the environmental controller can receive [range: 0 ~ 2147483647, default: 20000]
```
Comment (brachy84) [inline config/rftools/rftools.cfg]: and this

File: config/rftools/rftools.cfg [hunk 14]
```diff
@@ -1549,7 +1549,7 @@ screen {
     # Maximum RF storage that the screen controller can hold [range: 0 ~ 2147483647, default: 60000]
     I:screenControllerMaxRF=60000
 
-    # RF per tick that the the screen controller can receive [range: 0 ~ 2147483647, default: 1000]
```
Comment (brachy84) [inline config/rftools/rftools.cfg]: and this

File: config/rftools/rftools.cfg [hunk 15]
```diff
@@ -1923,7 +1923,7 @@ teleporter {
     # Maximum RF storage that the charged porter item can hold (note that teleporting this way uses 50% more RF then with a matter transmitter) [range: 0 ~ 2147483647, default: 200000]
     I:chargedPorterMaxRF=200000
 
-    # RF per tick that the the charged porter item can receive [range: 0 ~ 2147483647, default: 2000]
```
Comment (brachy84) [inline config/rftools/rftools.cfg]: and this

**Review by brachy84 — CHANGES_REQUESTED** [review]
After some testing I found out that configs get reloaded. I'm not sure if changed values get reloaded, but the comments get reloaded. Please remove the changes on config comments

File: config/exnihilocreatio/ExNihiloCreatio.cfg (hunk 1 above)
Comment (hjae78) [reply config/exnihilocreatio/ExNihiloCreatio.cfg]: How come?

Comment (brachy84) [reply config/exnihilocreatio/ExNihiloCreatio.cfg]: as i said above. it reloads the file.

Comment (a-thoren) [reply config/exnihilocreatio/ExNihiloCreatio.cfg:116]: Ah okay, didn't see that comment

Comment (CrafterKolyan) [reply config/exnihilocreatio/ExNihiloCreatio.cfg]: Reverted all cfg files

**Review by brachy84 — APPROVED** [review]

**Pull request merged**